conf/airframes/TUDELFT/tudelft_fan_demo.xml

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE airframe SYSTEM "../airframe.dtd">
+
+<airframe name="bebop_indi">
+
+  <firmware name="rotorcraft">
+    <target name="ap" board="bebop"/>
+
+    <target name="nps" board="pc">
+      <module name="fdm" type="jsbsim"/>
+      <module name="udp"/>
+    </target>
+
+    <!--define name="USE_SONAR" value="TRUE"/-->
+
+	<!-- Subsystem section -->
+    <module name="telemetry" type="transparent_udp"/>
+    <module name="radio_control" type="datalink"/>
+    <module name="actuators" type="bebop"/>
+    <module name="imu" type="bebop"/>
+    <module name="gps" type="datalink"/>
+    <module name="stabilization" type="indi">
+      <define name="INDI_RPM_FEEDBACK" value="TRUE"/>
+    </module>
+    <module name="stabilization" type="rate_indi"/>
+    <module name="ahrs" type="int_cmpl_quat">
+      <configure name="USE_MAGNETOMETER" value="FALSE"/>
+      <define name="AHRS_USE_GPS_HEADING" value="TRUE"/>
+    </module>
+    <module name="ins" type="extended"/>
+    <module name="guidance" type="indi">
+      <define name="GUIDANCE_INDI_SPECIFIC_FORCE_GAIN" value="-500.0"/>
+      <define name="GUIDANCE_INDI_THRUST_DYNAMICS" value="0.1"/>
+      <define name="GUIDANCE_INDI_RC_DEBUG" value="FALSE"/>
+      <define name="GUIDANCE_INDI_POS_GAIN" value="0.7"/>
+      <define name="GUIDANCE_INDI_SPEED_GAIN" value="1.5"/>
```

Comment:
Move these to a GUIDANCE_INDI section? 

The difference is that variables defined here are included in the make flags and should be reserved for defines that are used to enable/disable parts of the code from being compiled. If these are just settings, it is better to place them in a <section>, these are included in generated/airframe.h. This distinction helps to reinforce how these parameters will be used in the code.